chapters/en/slides/module1/module1_03.Rmd

```diff
@@ -0,0 +1,253 @@
+---
+params:
+  dynamictitle: "module1_03"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--atx-headers"
+---
+
+```{r include=FALSE}
+# Update the title in the YAML preamble for each slide show
+knitr::opts_chunk$set(
+    echo = TRUE,
+    base.dir = ".",
+    base.url = "/",
+    fig.path = paste("../../../../static/module1/", params$dynamictitle,"/", sep = ""))
+
+source('../../../../slide-setup.R')
+```
+
+type: slides
+
+# Lines and layers
+
+Notes:
+In this slide deck we will see how to create lineplots
+and how to combine line and point plots together
+via layers in Altair.
+
+---
+
+## Visualizing trends over time with scatter plots is not ideal
+
+```{python}
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+
+alt.Chart(cars).mark_point().encode(
+    x='Year',
+    y='Miles_per_Gallon')
+```
+
+Notes:
```

Comment:
Again, we need to motivate with a compelling case and story.

Reply:
I have rewritten to have more of a narrative throughout this slide deck.